spec/support/platform_helpers.rb

```diff
@@ -6,15 +6,15 @@ module PlatformHelpers
   # Ruby runtime engines
 
   def mri?
-    RUBY_ENGINE == 'ruby'.freeze
+    'ruby'.freeze == RUBY_ENGINE
```

Comment:
I think this is a bug of the https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/YodaCondition cop.
And I believe it happens because it doesn't know that `#freeze` is not really a complex method call, and `'ruby'.freeze` is still an immediate value.

Reply:
For this file, can I suggest adding `# frozen_string_literal: true` to the top, and reverting the YodaCondition changes?

Reply:
sure!